css/style.css

```diff
@@ -6,14 +6,165 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+header img{
+    width:50px;
+    height:50px;
+}
+header{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+/*nav*/
+.topnav a:hover, .topnav a:focus{
+    color:rgb(255, 123, 0);
+}
+.topnav {
+    overflow: hidden;
+}
+.topnav a {
+    float: left;
+    display: block;
+    color: grey;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+.topnav .icon {
+    display: none;
+  }
+  
+  
+
+/*main*/
+.intro{
+    background-image: url(/img/first-background.jpg);
+    height:500px;
+    color: white;
+    text-align: center;
+    padding-top: 150px;
+}
+.intro h1{
+    font-weight: 100;
+    font-size: 3.5rem;
+}
+.intro h2{
+    font-weight: 50;
+}
+.intro button{
+    background-color: rgb(255, 123, 0);
+    border:0;
+    border-radius: 5%;
+    padding:15px;
+    margin-top: 20px;
+}
+
+.container{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.container img{
+    width: 90%;
+    height:90%;
+}
+.items{
+    text-align: center;
+    margin: 15%;
+    margin-top: 5%;
+    margin-bottom:5%;
+    font-size: 1rem;
+    font-weight: 400;
+}
+.items h2{
+    font-weight: 300;
+    font-size: 2rem;
+}
+
+
+/*footer*/
+footer{
+    text-align: center;
+}
+footer::before{
+
+}
```

Comment:
```suggestion
```
This rule is empty so we should delete it. Software teams generally don't leave in empty code